contracts/Membership.sol

```diff
@@ -153,7 +171,9 @@ function test() public {
     }
 
     function removeAdmissionToken(uint256 index) public onlyOwner {
-        require(admissionTokens.length > index, "index overflow");
+        if (admissionTokens.length <= index) {
```

Comment:
After discuss with @ayame30 , It is better to add index in admissionTokenSetting, to handle removing element in AdmissionToken Address[]